indico/htdocs/js/indico/RegistrationForm/section.js

```diff
@@ -705,8 +705,9 @@ ndRegForm.directive("ndSocialEventSection", function() {
                            align: 'center',
                            width: 50,
                            editable: true,
-                           edittype: "text",
+                           edittype: 'text',
                            editoptions: {size: "7", maxlength: "20"},
+                           pattern: '/^(\\d*[.])?\\d+$/'
```

Comment:
Might be better if we don't force to type 2 numbers after the decimal point ^(\\d+(\\.\\d{1,2})?)?$`